src/qibolab/instruments/qblox/controller.py

```diff
@@ -97,6 +97,22 @@ def disconnect(self):
             self.cluster.disconnect()
             self.is_connected = False
 
+    def _get_module_channel_map(self, module_name: str, qubits: dict):
+        """Retrieve all the channels connected to a specific Qblox module.
+
+        This method updates the `channel_port_map` attribute of the specified Qblox module
```

Comment:
I introduced this function mainly to set up the attribute `self.channel_port_map`, otherwise every time (e.g. inside the `get_if() ` function) is required the information "which channel is connected to which port" 2 for cycles are needed to extract this from the qubits objects. Instead in this way the cycles are performed once for all and the information is stored in the old attribute `self.channel_port_map`